2022/_data/talks.json

```diff
@@ -1054,7 +1051,6 @@
       "id": "627c1bdd26f9720858a38ef7",
       "title": "The ups and downs of deploying AI in the wild",
```

Comment:
and this one, as well...

Reply:
The error is the metadata, should be "eng"